### PR TITLE
Add instructions for external contributors

### PR DESCRIPTION
**Pull request by ptrdvrk**
The CONTRIBUTING.md references AppLand Contributor License Agreement and Code of Conduct.

### CODE REVIEW

**Review by apotterri — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Contributing to appmap-cli
+
+We are incredibly thankful for the contributions we receive from the community. Before contributing, please take a moment to read our [Contributor License Agreement](https://github.com/applandorg/community/blob/master/docs/CLA%20Instructions.pdf) and our [Code of Conduct](https://github.com/applandorg/community/blob/master/docs/Code%20of%20Conduct%20for%20Contributors.pdf).
+
+## Contributor License Agreement
+We require our external contributors to sign a Contributor License Agreement ("CLA") in order to ensure that
+our projects remain licensed under Free and Open Source licenses such as while allowing
+Appland to build a sustainable business.
```
Comment (kgilpin) [inline CONTRIBUTING.md:8]: AppLand not Appland

Comment (apotterri) [reply CONTRIBUTING.md:8]: Fixed.

**Comment by kgilpin** [comment]
Already merged though :-(

**Comment by apotterri** [comment]
Yeah, I fixed it on master.

**Comment by ptrdvrk** [comment]
Thank you @apotterri @kgilpin . The "Appland" spelling is used in the CLA instructions and in the legal agreement language so I kept it. I'll ask Elizabeth if we need a version with correct spelling.

**Pull request merged**